packages/examples/cvat/exchange-oracle/src/endpoints/filtering.py

```diff
@@ -131,13 +133,19 @@ def _selectable_fields(self) -> list[str] | None:
         )
         return list(selectable_fields_enum.__members__)
 
+    @classmethod
+    def get_default_field_value(cls, field_name: str) -> Any:
+        default_value = cls.model_fields[field_name].get_default()
+        if isinstance(default_value, fastapi.params.Query):
```

Comment:
Probably, it's better to check for `params.FieldInfo` instead for better compatibility with other possible field declarations.

Reply:
```suggestion
        if isinstance(default_value, fastapi.params.FieldInfo):
```

Reply:
done